content/docs/conditional-rendering.md

```diff
@@ -186,13 +187,13 @@ render() {
 }
 ```
 
-Just like in JavaScript, it is up to you to choose an appropriate style based on what you and your team consider more readable. Also remember that whenever conditions become too complex, it might be a good time to [extract a component](/docs/components-and-props.html#extracting-components).
+जैसे की जावास्क्रिप्ट में होता हे, यह हम पे निर्भर करता हे की कोनसा स्टाइल उसे करे जिससे आप और आपि टीम को समझना आसान हो. यह भी याद रहना की जैसे कंडीशंस बड़ी और बेचिदा हो, तब कौम्पोनॅन्ट को एक्सट्रेक्ट करना अच्छा रहता हे [कौम्पोनॅन्ट को एक्सट्रेक्ट](/docs/components-and-props.html#extracting-components)।
```

Comment:
`हे` => `है`

Reply:
done

Reply:
`हो.` => `हो।`

Reply:
More of these are there

`हे` => `है`

I will suggest use some IDE and do CTRL+F and replace all this will update it everywhere

Reply:
done

Reply:
I still see this at two places.
Line 26 `रहे हे।`
Line 74 ` बना रहे हे`

Can we make the fixes for these two also

Reply:
Yeah